io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,42 @@
+{
+	"id": "io.github.torbuntu.Leikr",
+	"branch": "1.0",
```

Comment:
this needs to be "stable" for flathub, but you should probably leave it as unset and the build process will do it for you..